lib/src/configuration.dart

```diff
@@ -74,10 +78,10 @@ abstract class Configuration {
   static late String defaultRealmPath = _path.join(defaultStoragePath, 'default.realm');
 
   Configuration._(
-    List<SchemaObject> schemaObjects, {
+    this.schemaObjects, {
```

Comment:
Is this still always required? Aren't we loading the schema from the realm file if the schema is not defined?

Reply:
I opted to keep it required since loading the schema from file is the more uncommon use case.